modules/led.sh

```diff
@@ -91,6 +94,41 @@ function dance {
   led green "$current_green"
 }
 
+function new_year {
+
+  current_green=$(led "green")
+  current_red=$(led "red")
+
+  set_brightness 0 0 && set_brightness 1 0
+  sleep 1
+
+  counter=0
+  while [ $counter -le 2 ]
+  do
+    set_brightness 1 0 && set_brightness 0 1
+    sleep 0.5
+    set_brightness 1 1 && set_brightness 0 0
+    sleep 0.5
+    counter=$(( counter + 1 ))
+  done
+
+  set_brightness 1 0 && set_brightness 0 0
+  sleep 0.5
+
+  counter=0
+  while [ $counter -le 2 ]
+  do
+    set_brightness 1 1 && set_brightness 0 1
+    sleep 0.5
+    set_brightness 1 0 && set_brightness 0 0
+    sleep 0.5
+    counter=$(( counter + 1 ))
+  done
+
+  led green "$current_green"
+  led red "$current_red"
+}
+
 function led_help {
   echo ""
   echo "Usage: $(basename "$0") led [green|red|dance] [mode]"
```

Comment:
Include `new_year` in the `led_help` function.
`echo "Usage: $(basename "$0") led [green|red|dance] [mode]"` -> `echo "Usage: $(basename "$0") led [green|red|dance|new_year] [mode]"`

Maybe consider adding it to the examples below as well?